tests/test_sanity.py

```diff
@@ -12,6 +13,7 @@ def test_webserver_response_code():
     assert response.status_code == 200
 
 
+@pytest.mark.skip(reason="We dont have mysql server just yet")
```

Comment:
Remove this test all together. We do not need code which is not doing something relevant.

Reply:
Done